idea-plugin/src/main/java/com/palantir/javaformat/intellij/PalantirJavaFormatFormattingService.java

```diff
@@ -89,9 +93,20 @@ public void run() {
             }
 
             try {
-                String formattedText = applyReplacements(
-                        request.getDocumentText(),
-                        formatterService.get().getFormatReplacements(request.getDocumentText(), toRanges(request)));
+                logger.info(String.format(
+                        "Received request to format file=%s, length=%s with ranges=%s",
+                        Optional.ofNullable(request.getIOFile())
+                                .map(file -> file.toPath().toString())
+                                .orElse("null"),
+                        request.getDocumentText().length(),
+                        request.getFormattingRanges()));
+                List<Replacement> replacements =
+                        formatterService.get().getFormatReplacements(request.getDocumentText(), toRanges(request));
+                logger.debug(String.format(
```

Comment:
is it worth surrounding this in a logger.isDebugEnabled() check (or whatever the call is) just so it doesn't do all the formatting if debug isn't on?